noir-projects/aztec-nr/aztec/src/discovery/partial_notes.nr

```diff
@@ -24,8 +25,9 @@ pub global MAX_PARTIAL_NOTE_PRIVATE_PACKED_LEN: u32 =
     MAX_LOG_CONTENT_LEN - PARTIAL_NOTE_PRIVATE_LOG_CONTENT_NON_NOTE_FIELDS_LEN;
 
 /// The slot in the PXE capsules where we store a `CapsuleArray` of `DeliveredPendingPartialNote`.
-// TODO(#11630): come up with some sort of slot allocation scheme.
-pub global DELIVERED_PENDING_PARTIAL_NOTE_ARRAY_LENGTH_CAPSULES_SLOT: Field = 77;
+pub global DELIVERED_PENDING_PARTIAL_NOTE_ARRAY_LENGTH_CAPSULES_SLOT: Field = sha256_to_field(
+    "DELIVERED_PENDING_PARTIAL_NOTE_ARRAY_LENGTH_CAPSULES_SLOT".as_bytes(),
```

Comment:
y u no review request

Two comments:
a) let's add a domain separator to avoid having to think about users using capsules, e.g. `hash(["aztec-nr", name])`
b) can this be made comptime? i guess this is perhaps not a huge issue since capsules are unconstrained, so we should never need to compute these values in circuits

Reply:
b) globals are always comptime

Reply:
@vezenovm can you please confirm that I am not lying here?

Reply:
Yes, globals are all computed at comptime